sebastian-rendon/tarea-2/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="Mock Magazine Cover about Oscar Piastri"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Magazine</title>
+  </head>
+  <body>
+    <main class="main__magazine">
+      <span
+        title="Edition: September 4, 2022 - 42th Edition"
+        class="edition__details"
+      >
+        Sept. 4, 2022 - Ed. 42
+      </span>
+      <header class="header">
+        <h1 title="Checkered flag" class="header__magazine-title">
+          <span>checkered</span>
+          <span>flag</span>
+        </h1>
+      </header>
+      <img
```

Comment:
@s-rendong you should figure tag for all images

Reply:
fixed with https://github.com/emolinaab/bootcamp-001/pull/88/commits/8393c4295e96ea5b6bb1fd2804e2903da42f3180